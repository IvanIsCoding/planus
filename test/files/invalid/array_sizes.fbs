struct Foo {
  x: [bool: -1];
  y: [bool: 99999999999999999];
}