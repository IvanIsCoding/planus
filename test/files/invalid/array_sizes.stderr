[0m[1m[38;5;9merror[0m[1m: size cannot be negative[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/array_sizes.fbs:2:13
  [0m[34mâ”‚[0m
[0m[34m2[0m [0m[34mâ”‚[0m   x: [bool: [0m[31m-1[0m];
  [0m[34mâ”‚[0m             [0m[31m^^[0m

[0m[1m[38;5;9merror[0m[1m: size is too large[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/array_sizes.fbs:3:13
  [0m[34mâ”‚[0m
[0m[34m3[0m [0m[34mâ”‚[0m   y: [bool: [0m[31m99999999999999999[0m];
  [0m[34mâ”‚[0m             [0m[31m^^^^^^^^^^^^^^^^^[0m

[0m[1m[38;5;9merror[0m[1m: Arrays are not currently supported in planus[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/array_sizes.fbs:2:6
  [0m[34mâ”‚[0m
[0m[34m2[0m [0m[34mâ”‚[0m   x: [0m[31m[bool: -1][0m;
  [0m[34mâ”‚[0m      [0m[31m^^^^^^^^^^[0m

[0m[1m[38;5;9merror[0m[1m: Arrays are not currently supported in planus[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/array_sizes.fbs:3:6
  [0m[34mâ”‚[0m
[0m[34m3[0m [0m[34mâ”‚[0m   y: [0m[31m[bool: 99999999999999999][0m;
  [0m[34mâ”‚[0m      [0m[31m^^^^^^^^^^^^^^^^^^^^^^^^^[0m

Error: Bailing because of errors
